demo/admin/src/common/routes/MasterHeader.tsx

```diff
@@ -1,7 +1,7 @@
 import { BuildEntry, Header, UserHeaderItem } from "@comet/cms-admin";
 import * as React from "react";
 
-import { ContentScopeControls } from "./ContentScopeProvider";
+import { ContentScopeControls } from "../ContentScopeProvider";
```

Comment:
why is the header moved into routes? do you think it belongs there?

Reply:
Actually just because of this line, which makes the MasterMenu and MasterHeader a Sub-Component of Routes: https://github.com/vivid-planet/comet/pull/1262/files/f876b9f2d83f01352d8ca5d971f12d1f6201403d#diff-5f9d32680b749339947cbf88c3d0577cf7e0f2ac986df1b08d7dbc5537250011R21

But no, it doesn't really make sense since the ContentScopeControls should also be moved here then. I think I will change it back.

Reply:
I restructured the files.